src/yamlfix/entrypoints/cli.py

```diff
@@ -90,11 +90,11 @@ def cli(  # pylint: disable=too-many-arguments
 
     Use - to read from stdin. No other files can be specified in this case.
     """
-    files_to_fix: List[TextIOWrapper] = []
+    files_to_fix: Files = []
     if "-" in files:
         if len(files) > 1:
             raise ValueError("Cannot specify '-' and other files at the same time.")
-        files_to_fix = [sys.stdin]
+        files_to_fix = (sys.stdin,)  # type: ignore[assignment]
```

Comment:
`Files` must be a tuple of `TextIOWrapper` (or a list of strings), so in this case I've turned into a tuple, however, `sys.stdin`'s type is incompatible with `TextIOWrapper`, hence the `ignore` for now.